docs/user_guide/assessment.rst

```diff
@@ -1,21 +1,128 @@
+harms are not mutually exclusive. For more information, review 
+Fairlearn's `2021 SciPy tutorial <https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf>`_.
+
+2. Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. currentmodule:: fairlearn.metrics
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular use case or deployment context. For example, for 
+speech-to-text transcription, this might include groups who speak a regional dialect or people who are a  
+native or a non-native speaker.
 
-The :py:mod:`fairlearn.metrics` module provides the means to assess fairness-related
-metrics for models. This applies for any kind of model that users may already
-use, but also for models created with mitigation techniques from the
-:ref:`mitigation` section. The :ref:`dashboard` provides a visual way to
-compare metrics between models as well as compare metrics for different groups
-on a single model.
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
 
-Ungrouped Metrics
+3. Quantify harms
 ^^^^^^^^^^^^^^^^^
 
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are "qualified"). One possible metric is the false negative rate: fraction of qualified candidates that are screened out. Note that before we attempt to classify candidates, we need to determine the construct validity of the "qualified" status; more information on construct validity can be found in :ref:`construct_validity`.
+
+* For a speech-to-text application, the harm could be measured by disparities in the word error rate for different group, measured by the number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, harms might not be readily quantifiable. 
+In these cases, we might choose to use a closely related variable,
+often called a "proxy",  
+to stand in for the missing variable. For example, suppose that in 
+the job screening scenario, we have data on whether the candidate passes 
+the first two stages, but not if they are ultimately recommended for the 
+job.
+
+As an alternative to the unobservable final recommendation, we could
+therefore measure the harm using the proxy variable indicating whether
+the candidate passes the first stage of the screen.
+If you choose to use a proxy variable to 
+represent the harm, check the proxy variable regularly to ensure it 
+remains useful over time.
```

Comment:
Optional, but we might want to link to our section of the User Guide about [construct validity](https://fairlearn.org/v0.7.0/user_guide/fairness_in_machine_learning.html#what-is-construct-validity) here since it's a framework for assessing the "usefulness" of a proxy variable.

Reply:
Done!